src/RequestParser.cpp

```diff
@@ -282,6 +283,10 @@ void RequestParser::parseHttpRequest(const std::string& requestString, HTTPReque
 	requestLine = parseVersion(requestLine, request);
 	checkForCRLF(requestLine, request);
 
+    LOG_DEBUG << "Parsed method: " << request.method;
```

Comment:
Since method is an enum, we could make an ostream overload to print out the string for the method

Reply:
Added insertion overload for enum method.

Reply:
LogData doesn't seem like the right place, since it already has a templated ostream receiver: if there is an ostream overload, LogData can use it.

In #39 I implemented it like the existing ostream insertion operators for Config file and renamed the file LogConfigFile.cpp to reflect that it contains different ostream inserters

Reply:
You mean #38, not #39, right? 

Alright, if you created a general file for all ostream overloads I will remove the last commit and merge directly. The new file will be included via the LogData.hpp after your PR is merged.

Reply:
Need this conversation resolved for merge.

Reply:
Yes you're right: #38